homeassistant/components/media_player/apple_tv.py

```diff
@@ -32,8 +32,8 @@
 DATA_APPLE_TV = 'apple_tv'
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_HOST): cv.string,
-    vol.Required(CONF_LOGIN_ID): cv.string,
+    vol.Optional(CONF_HOST): cv.string,
```

Comment:
These should still be `vol.Required` because this is the validation schema for when you set it up via configuration.